src/test/java/com/nirima/jenkins/plugins/docker/DockerTemplateTest.java

```diff
@@ -30,6 +30,7 @@
     String environmentsString = "environmentString";
     String hostname = "hostname";
     String user = "user1";
+    String extraGroupsString = "foo\nbar";
```

Comment:
This isn't a comment about this line, it's just close-by...

We should also have unit tests added to `DockerTemplateBaseTest` that verify that
1) fillContainerConfig does _not_ set any "extra groups", not even setting things to null/empty, if the user hasn't given any extra groups.
2) fillContainerConfig _does_ set the extra groups that the user specified.
See `testFillContainerShmSize` and the methods that call it for an example.